api/src/main/java/net/kyori/adventure/sound/Sound.java

```diff
@@ -32,8 +32,18 @@
 import static java.util.Objects.requireNonNull;
 
 /**
- * A sound.
+ * Represents an in-game sound which can be played to the client.
  *
+ * <p>A sound consists of:</p>
+ * <ul>
+ *   <li>key/type - the resource location of this sound (e.g minecraft:ambient.cave or my_plugin:custom_sound</li>
+ *   <li>source - a {@link Source} telling the game where the sound is coming from</li>
+ *   <li>volume - a number in the range [0,-&#62;) representing how loud the sound should be played,
```

Comment:
Yeah, that's intended, "->)" would imply up to but not including infinity. Should it be written differently?

Reply:
the standard notation that I've seen for that is `[0,∞)` -- the `)` indicates it's an exclusive range (since it is not possible to have an inclusive upper bound if the upper bound is non-finite iirc)